src/porepy/geometry/bounding_box.py

```diff
@@ -138,3 +139,22 @@ def make_bounding_planes_from_box(box: dict[str, float]) -> list[np.ndarray]:
 
     bound_planes = [west, east, south, north, bottom, top]
     return bound_planes
+
+
+class DomainSides(NamedTuple):
```

Comment:
I think a very similar structure can be used for FractureNetwork classes as well. It may be that we should change the order of the parameters in the named tuple, to that end.

Not this PR, but feel free to stop by at some point.